coveo-stew/README.md

```diff
@@ -103,86 +183,75 @@ mypy = { set-config = False }
 mypy = { set-config = "mypy.ini" }
 ```
 
+See https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml
 
-## `stew build`
-
-Store the project and its **locked dependencies** to disk. 
+### pytest
 
-Optimally used to create truly repeatable builds and workflows (e.g.: docker images, terraform, S3, puppet...)
+Pytest is configured to run with `--doctest-modules --tb=short --durations=5` as well as JUnit report generation.
 
-The folder can later be installed offline with `pip install --no-index --find-links <folder> <project-name>`
+Some additional options are available:
```

Comment:
Are the two options below the only two that we can add for pytest? The way this is worded, it seems that we can'T we use this section to pass other arguments to pytest? For example, I always run with -s -v. Whenever I switch to poetry (happening sonner than later), I'd probably like to continue to do so.

Reply:
You're right, I started implementing something so that all runners may accept custom arguments.